connectivity/cellular/source/framework/RIL/RIL_CellularContext.cpp

```diff
@@ -0,0 +1,1164 @@
+                                                               _data_call_addresses->_local_addr_ipv6,
+                                                               sizeof(_data_call_addresses->_local_addr_ipv6),
+                                                               temp, sizeof(temp));
+        mbed_cellular_util::separate_ip_addresses(response->addresses,
+                                                               _data_call_addresses->_local_addr_ipv4,
+                                                               sizeof(_data_call_addresses->_local_addr_ipv4),
+                                                               temp, sizeof(temp));
+
+        tr_debug("request_setup_data_call_response, addresses: %s, IPv4: %s, IPv6: %s",
+                 response->addresses, _data_call_addresses->_local_addr_ipv4, _data_call_addresses->_local_addr_ipv6);
+    } else {
+        return NSAPI_ERROR_DEVICE_ERROR;
+    }
+
+    if (response->dnses) {
+        mbed_cellular_util::separate_ip_addresses(response->dnses,
+                                                               _data_call_addresses->_primary_dns_addr_ipv6,
+                                                               sizeof(_data_call_addresses->_primary_dns_addr_ipv6),
+                                                               _data_call_addresses->_secondary_dns_addr_ipv6,
+                                                               sizeof(_data_call_addresses->_secondary_dns_addr_ipv6));
+
+        mbed_cellular_util::separate_ip_addresses(response->dnses,
+                                                               _data_call_addresses->_primary_dns_addr_ipv4,
+                                                               sizeof(_data_call_addresses->_primary_dns_addr_ipv4),
+                                                               _data_call_addresses->_secondary_dns_addr_ipv4,
+                                                               sizeof(_data_call_addresses->_secondary_dns_addr_ipv4));
+
+        tr_debug("request_setup_data_call_response, DNSes: %s", response->dnses);
+        tr_debug("request_setup_data_call_response, DNSes IPv4: %s, %s",
+                 _data_call_addresses->_primary_dns_addr_ipv4, _data_call_addresses->_secondary_dns_addr_ipv4);
+        tr_debug("request_setup_data_call_response, DNSes IPv6: %s, %s",
+                 _data_call_addresses->_primary_dns_addr_ipv6, _data_call_addresses->_secondary_dns_addr_ipv6);
+    }
+
+    if (response->gateways) {
+        mbed_cellular_util::separate_ip_addresses(response->gateways,
+                                                               _data_call_addresses->_gateway_addr_ipv6,
+                                                               sizeof(_data_call_addresses->_gateway_addr_ipv6),
+                                                               temp, sizeof(temp));
+        mbed_cellular_util::separate_ip_addresses(response->gateways,
```

Comment:
`mbed_cellular_util::separate_space_delimited_addresses(false, response->gateways,`

Reply:
the same as above